### ISSUE TITLE
Feature Request - "Close ALL" on tab context menu

### ISSUE THREAD

**Issue by LagunaJim** [notepad-plus-plus/notepad-plus-plus#3411]
Currently there are options to Close to the left, Close to the Right, and CLose all but this one..

I would love to see "Close All" as a choice as well -- I love the feature that reopens previous tabs when I launch, but when this needs cleanup currently I have to go into the File menu to find "Close All".  Would rather just right-click and zap!

Thanks!

**Comment by vlakoff** [comment]
Previously, I was thinking the same as you, but I've changed my mind:

Doing "Close all but this one" then closing the remaining tab by a middle click isn't that tedious, and I think it's better than adding an item to an already heavy menu.

Also note your requested feature can be "emulated" as I described, whereas other commands ("Close to left", "Close this one", etc.) cannot be emulated likewise.

Finally, but it's a different subject, it would be great to be able to customize this menu, like it's already possible for the edit zone context menu.
